metale_amorficzne/description/01_general.md

```diff
@@ -1,7 +1,272 @@
+  from 25mN to 200mN)
+
+---
+
+## Methods and Results
+
+Initially I tried classification, but after a while of trials with decision
+trees, I realised that in this case clustering should be performed, as the data
+in unlabelled.
+
+### Hierarchical clustering
+
+Hierarchical clustering gave the best results.
+
+### KMeans clustering
+
+### ChatGPT clustering
+
+### OPTICS clustering
+
+## Discussion
+
+---
+
+### References
+
+1. Hay, J.L.; Pharr, G.M. (2000). "Instrumented indentation testing". ASM
+Handbook, Vol. 8, Mechanical Testing and Evaluation. ASM International. pp.
+231–242. ISBN 978-0-87170-389-7.
+2. W.C. Oliver and G.M. Pharr (2011). "Measurement of hardness and elastic
+modulus by instrumented indentation: Advances in understanding and refinements
+to methodology" (PDF). Journal of Materials Research.
+3. Nanoindentation Lecture 1 Basic Principle, by Do Kyung Kim, Dept. of Material
+Science and Engineering KAIST, Korea.
+4. Fischer-Cripps, A.C. Nanoindentation. (Springer: New York), 2004.
+5. W.C. Oliver and G.M. Pharr. An improved technique for determining hardness
+and elastic modulus using load and displacement sensing indentation experiments,
+J. Mater. Res., vol. 7, No. 6, June 1992.
+6. Peker, A.; Johnson, W. L. (25 October 1993). "A highly processable metallic
+glass: Zr41.2Ti13.8Cu12.5Ni10.0Be22.5" (PDF). Applied Physics Letters. 63 (17):
+2342–2344. Bibcode:1993ApPhL..63.2342P. doi:10.1063/1.110520.
+7. Nishiyama, Nobuyuki; Amiya, Kenji; Inoue, Akihisa (October 2007). "Novel
+applications of bulk metallic glass for industrial products". Journal of
+Non-Crystalline Solids. 353 (32–40): 3615–3621. Bibcode:2007JNCS..353.3615N.
+doi:10.1016/j.jnoncrysol.2007.05.170.
+8. Nishiyama, N.; Amiya, K.; Inoue, A. (March 2007). "Recent progress of bulk
+metallic glasses for strain-sensing devices". Materials Science and Engineering:
+A. 449–451: 79–83. doi:10.1016/j.msea.2006.02.384.
+9. Saotome, Y.; Iwazaki, H. (2000). "Superplastic extrusion of microgear shaft
+of 10 μm in module". Microsystem Technologies. 6 (4): 126.
+doi:10.1007/s005420050180. S2CID 137549527.
+10. Kumar, G.; Tang, H. X.; Schroers, J. (2009). "Nanomoulding with amorphous
+metals". Nature. 457 (7231): 868–872. Bibcode:2009Natur.457..868K.
+doi:10.1038/nature07718. PMID 19212407. S2CID 4337794.
+11. Hojati-Talemi, Pejman (2011). "High performance bulk metallic glass/carbon
+nanotube composite cathodes for electron field emission". Applied Physics
+Letters. 99 (19): 194104. Bibcode:2011ApPhL..99s4104H. doi:10.1063/1.3659898.
+12. Maruyama, Masaaki (June 11, 2009). "Japanese Universities Develop Ti-based
+Metallic Glass for Artificial Finger Joint". Tech-on. "Fixing bones with
+dissolvable glass". Institute of Physics. October 1, 2009.
+13. "Engineers Develop Record-Breaking Steel". Engineering.com. Retrieved
+2022-06-24.
+14. "Record-breaking steel could be used for body armor, shields for
+satellites". jacobsschool.ucsd.edu. Retrieved 2022-06-24.
```

Comment:
1. Properly reformat. When wrapping lines in enumeration, wrapped lines should be indented with three spaces.
2. Use italics where applicable (similar to how it is on Wikipedia).
3. Fill in the missing links.